redesign-project/src/index.html

```diff
@@ -0,0 +1,67 @@
+---
+layout: layouts/default.njk
+title: Emma's Tea Spot
+---
+
+    {% include "_header.html" %}
+
+  <img src="https://lh3.googleusercontent.com/hrINEylFQ9GAW_WOJNL0CnnS-X9r8mBvemQsA1i79jzu2jGOtCy1SggNSjMG3rXaOc9tcT5RguefLjNLgsT1_6K7Ea98YyD7p1mDl7i9Z5Ti7Jzt6Kec1hKcXsFpA3IlKh2AmxX97d1_8dm9atd6RfamurWMW1Y41eWGPODkni5CbFYrTy5gM4TEFDuFNrl2YIxfICglgY7v9AgPQo5-Ys6Oyspl1662q9FP4U7v7CdLab5s-ivL8vhwz3URf6MKcspsnyNizA4tQUBajTJKneSaiV8XKaApDi7Gxh2qAM53WB8xV93GlTJ8ldCjeYx5P41yIswbUHVbIWogzY-1LYzKCHixpA5A3n8nIbZoLHbd071VyFZvjfdEClSKEUCGGib-_WaTheJvKM9-6CIG6-vtJBQya4oDK0xATSTvrFxQmtTsG0VmJta_hNw3V_VMeVq0TNwsn2EFLxYhf-GObboKpuTsNb6Lva7BjQBA4E12iqDvE66CHEB76zll0QtW4eU1CdvBws8USji28PNwebSjor9Af6oH0m14-u7z6-a4vaSaG-vB8vQwo6LH_133tT_Dukhn3NUFg442hQvQOHnOEX6tcMMh0OodNuhduTks_vJv-Iwcx5Pd2Mv5KMENi13uNAPTnSYcR0RQbsVz5PT9XouhBjW30iQqZtXPXM8E4Kn9j8jEiaLVQh3vfpd0lhqw3RL_GcfSt3ku=w882-h1344-no"
```

Comment:
Looks like you have two image files here -- is that intentional?
Also, remember to download these images and include them inside your `src` directory in an `images` directory and reference them from there vs hotlinking to an external source, please! 🙏